libr/include/r_anal.h

```diff
@@ -1955,13 +1957,12 @@ R_API RAnalCaseOp* r_anal_switch_op_add_case(RAnalSwitchOp * swop, ut64 addr, ut
 R_API RAnalCycleFrame* r_anal_cycle_frame_new (void);
 R_API void r_anal_cycle_frame_free (RAnalCycleFrame *cf);
 
-
 /* labels */
```

Comment:
stupid question, but... What are exactly function labels? Would it make sense to have a small description here?

Reply:
they are like flags but local to the function, like the labels starting with dollars in the gnu assembler. this was a request from xvilka many years ago. he can provide more examples and details

Reply:
beach is a label